src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -724,23 +699,12 @@ var ReactCompositeComponentMixin = {
           // top-level render calls, so I'm abstracting it away into
           // a function to minimize refactoring in the future
           var addendum = getDeclarationErrorAddendum(this);
-
-          if (location === ReactPropTypeLocations.prop) {
-            // Preface gives us something to blacklist in warning module
-            warning(
-              false,
-              'Failed Composite propType: %s%s',
-              error.message,
-              addendum
-            );
-          } else {
-            warning(
-              false,
-              'Failed Context Types: %s%s',
-              error.message,
-              addendum
-            );
-          }
+          warning(
+            false,
+            'Failed Context Types: %s%s',
+            error.message,
+            addendum
```

Comment:
Do you mind switching this to use getStackAddendumByID to show the stack? That can be a separate PR if you prefer.